### ISSUE TITLE
output not feed into the softmax

### ISSUE THREAD

**Issue by LilioISnotAfox** [researchmm/tasn#12]
Hello, I have noticed that in main_tasn.py, the outputs of fc directly are used to compute loss without feeding into the softmax, Is there any reason or just a mistake?
![image](https://user-images.githubusercontent.com/37403574/70792984-50db2480-1dd5-11ea-90ec-d6a767a2262c.png)

![image](https://user-images.githubusercontent.com/37403574/70792934-3e60eb00-1dd5-11ea-8ff8-2f254a1f67e5.png)

**Comment by Ahzyuan** [comment]
I think the 'nn.CrossEntropyLoss()' in the later procedure will apply softmax to the output

**Comment by xxx0090** [comment]
I also found it. Do you solve it?
